### ISSUE TITLE
Load-test and scale /user/breach-stats endpoint

### ISSUE THREAD

**Issue by groovecoder** [mozilla/blurts-server#1170]
When https://bugzilla.mozilla.org/show_bug.cgi?id=1559424 lands, we will get a *LOT* of traffic to `/user/breach-stats`. We should perform a load-test against it, and adjust as needed.

**Comment by groovecoder** [comment]
@mozilla/svcops - can we use the stage server for this?

**Comment by Micheletto** [comment]
> @mozilla/svcops - can we use the stage server for this?

@jbuck-mozilla-owner @jrgm we need FxA stage for some testing.

**Comment by jrgm** [comment]
@Micheletto our qa team is testing a pending release in stage, but they expected to be done by their EOD Monday 12th, but they are GMT+3, so Monday morning your time, you can fire away in stage.

**Comment by groovecoder** [comment]
Update here ...

I used the prod site because I needed to test with real FxA tokens that had the Monitor scope on them.

Erica reports ~3k distinct clients viewing the protection report per day in beta. Based on estimates from Flod, release has ~44x more clients than beta. This would translate to about 133k release clients per day viewing the report, and therefore hitting the `/user/breach-stats` endpoint, or roughly 1.5 req/s.

I ran a load test with locust that pushed about 5-10 req/s at the production endpoint and didn't see a linear climb in response times that would indicate performance problems that would scale with increased load.

![image](https://user-images.githubusercontent.com/71928/65188364-dd26b700-da33-11e9-8c70-5aebafbb0c9c.png)

(The 12-minute trend there continued for about 30-45m until our IP reputation monitoring service banned my IP from performing any more requests.)

So, I'm closing this out now.

So, I'm closing this ticket as having performed a successful load test.
